src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --active-link-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list :not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  display: inline-block;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  color: black;
+  text-decoration: none;
+}
+
+.nav_link.is-active {
+  position: relative;
+  color: var(--active-link-color);
+}
+
+.nav_link:hover,
+.nav_link:active {
+  color: var(--active-link-color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: inline-block;
+  width: 37px;
```

Comment:
you still have `width: 37px`
you need to add width for full container
if you move `is-active` class to some other larger item, your `after` element seems like:
<img width="395" alt="Screenshot 2023-01-17 at 12 50 31 PM" src="https://user-images.githubusercontent.com/60295799/212880166-00891bb1-2c35-4c23-875d-c79f91ed2a7f.png">